Documentation/security/policy/language.rst

```diff
@@ -827,7 +827,7 @@ latter rule will have no effect.
 .. note:: Layer 7 rules are not currently supported in `HostPolicies`, i.e.,
           policies that use :ref:`NodeSelector`.
 
-.. note:: Layer 7 policies --and pod annotations-- result in traffic being
+.. note:: Layer 7 policies result in traffic being
```

Comment:
Small suggestion, but not required: `Layer 7 policies will proxy traffic through an Envoy...`